convert2rhel/utils.py

```diff
@@ -66,6 +68,149 @@ class Color(object):
 BACKUP_DIR = os.path.join(TMP_DIR, "backup")
 
 
+# Code taken from https://stackoverflow.com/a/63773140
+class Process(multiprocessing.Process):
+    """Overrides the implementation of the multiprocessing.Process class."""
```

Comment:
Please explain briefly in the docstring why we need to override the standard library class.